frontend/views/components/Tooltip.vue

```diff
@@ -15,7 +15,7 @@ span.c-twrapper(
 
   .c-tooltip(
     :style='styles'
-    :class='{"has-text-center": isTextCenter, "is-active": isActive, manual, "is-dark-theme": $store.getters.isDarkTheme}'
+    :class='{"has-text-center": isTextCenter, "is-active": isActive, manual, "is-dark-theme": $store.getters.isDarkTheme,  "in-reduced-motion": isReducedMotionMode }'
```

Comment:
Tooltip needs a special treatment because, when it becomes active, it's appended to `<body>` not the root Vue component (`<div id='#app'>`)